src/framework/theme/components/autocomplete/autocomplete.component.ts

```diff
@@ -0,0 +1,154 @@
+/**
+ * @license
+ * Copyright Akveo. All Rights Reserved.
+ * Licensed under the MIT License. See License.txt in the project root for license information.
+ */
+
+// Component class scoped counter for aria attributes.
+let lastAutocompleteId: number = 0;
```

Comment:
I think declaration should go after imports

Reply:
Agree, replaced.